packages/wrangler-devtools/patches/0003-Support-previewing-subrequest-responses.patch

```diff
@@ -22,42 +22,42 @@ index c8aa624961..6201d73bba 100644
      let networkRequest: NetworkRequest|null|undefined = this.#requestsById.get(requestId);
      if (!networkRequest) {
        networkRequest = this.maybeAdoptMainResourceRequest(requestId);
-@@ -714,6 +714,11 @@ export class NetworkDispatcher implements ProtocolProxyApi.NetworkDispatcher {
+@@ -803,6 +803,11 @@ export class NetworkDispatcher implements ProtocolProxyApi.NetworkDispatcher {
        return;
      }
      this.getExtraInfoBuilder(requestId).finished();
 +    if (cfResponse !== undefined) {
 +      networkRequest.setContentDataProvider(async () => {
-+        return { error: null, content: cfResponse.body, encoded: cfResponse.base64Encoded };
++        return { error: '', content: cfResponse.body, encoded: cfResponse.base64Encoded };
```

Comment:
Why this change?

Reply:
This is a new type constraint in the frontend-devtools application.